internal/tui/components/keys.go

```diff
@@ -91,11 +95,15 @@ func (k KeyMap) ShortHelp() []key.Binding {
 // FullHelp returns keybindings for the expanded help view. It's part of the
 // key.Map interface.
 func (k KeyMap) FullHelp() [][]key.Binding {
+
+	tmpFollowMode := k.FollowMode
+	tmpFollowMode.SetHelp("f", "turn on/off follow mode")
```

Comment:
Remove this, not needed anymore

Reply:
Also, we can just say "toggle follow mode" instead of "on/off" - wherever we show the help text.